lib/route.js

```diff
@@ -152,8 +152,9 @@ function buildRouting (options) {
   }
 
   function hasRoute ({ options }) {
+    const normalizedMethod = options.method ? options.method.toUpperCase() : ''
```

Comment:
```suggestion
    const normalizedMethod = options.method?.toUpperCase() ?? ''
```

Reply:
Ah yes! I agree with you. I'll update as you suggested.